### PR TITLE
TimezoneConverter: fixes #4210 fixed error in timezone display

### LINKED ISSUE #4210: Timezone Converter: Incorrect and malformed values for NST and NDT
NST and NDT (according to Wikipedia, Newfoundland Standard Time and Daylight Time) should be UTC−03:30 and UTC−02:30 but display in Timezone Converter as "UTC-1:-49:-60" and "UTC+0:-49:-60".

------
IA Page:  http://duck.co/ia/view/timezone_converter 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @GlitchMr

### PR DESCRIPTION
**Pull request by AasthaGupta**
TimezoneConvertor: Fixed error in displaying timezone.The error was because it was taking modulo of negative number while separating it into minutes and hours in parse_timezone function.

Fixes #4210 

## People to be notified
@glitchmr

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/timezone_converter

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@AasthaGupta thanks for catching this -- can you please add a test that proves it works?

File: lib/DDG/Goodie/TimezoneConverter.pm [hunk 1]
```diff
@@ -46,7 +46,12 @@ sub parse_timezone {
     $minutes //= 0;
 
     my $hour = int( $timezones{$name} / 100 );
-    $minutes += $timezones{$name} % 100;
+    # To avoid modulo of negative numbers
+    my $m = abs( $timezones{$name} ) % 100;
+    if ( $timezones{$name} < 0 ){
```
Comment (moollaza) [inline lib/DDG/Goodie/TimezoneConverter.pm]: nit: we typically simply these like: `$m *= -1 if $timezones{$name} < 0;`

**Comment by mintsoft** [comment]
@moollaza LGTM now

**Comment by mintsoft** [comment]
Thanks for this @AasthaGupta !

**Pull request merged**